app/models/todo.rb

```diff
@@ -16,6 +16,7 @@ class Todo < ApplicationRecord
 
   scope :completed, -> { where completed: true }
   scope :active, -> { where completed: false }
+  scope :older_than_two_weeks, -> { where "created_at <= ?", 2.weeks.ago.to_datetime }
```

Comment:
Since we aren’t using this scope anywhere else, I am not sure there is a benefit it in adding it to the model vs just querying for it in the rake task. Wdyt?

Reply:
Definitely on the right track though! I would just add the query to the rake task and this should be good to go. Thanks for the PR 😁

Reply:
I agree with your request here, but would like to get this going today. I'll merge and we can tweak later if desired.